source/plugins/go/src/extension/extension.go

```diff
@@ -91,3 +118,35 @@ func getDataTypeToStreamIdMapping() (map[string]string, error) {
 	}
 	return datatypeOutputStreamMap, nil
 }
+
+func (e *Extension) IsContainerLogV2() bool {
+	extensionconfiglock.Lock()
+	defer extensionconfiglock.Unlock()
+	if len(e.dataCollectionSettings) > 0 && e.dataCollectionSettings["enableContainerLogV2"] != "" {
+		message := fmt.Sprintf("isContainerLogV2: %s", e.dataCollectionSettings["enableContainerLogV2"])
+		logger.Printf(message)

```

Comment:
This will be too much log messages since this log message for every record.  I think, you can remove the info log messages .

Reply:
removed